build.gradle.kts

```diff
@@ -62,7 +62,7 @@ spotless {
 
         removeUnusedImports()
 
-        palantirJavaFormat(libs.versions.palantir.get()).style("GOOGLE").formatJavadoc(true)
+        palantirJavaFormat().formatJavadoc(true)
```

Comment:
As this doesn't have anything to do with Evergreen, can we do a separate PR for it?

Reply:
yeah, will do.